### DISCUSSION TITLE
How to use the Date Picker?

### DISCUSSION THREAD

**Discussion by JF-626164** [Power-Components/livewire-powergrid#203] [General]
Hi all! I've spent most of the day trying to figure out how to get the makeInputDatePicker to work. I've found a few examples using 'created_at' for reference and all it simply shows is, `->makeInputDatePicker('created_at)` and that's apparently supposed to be it.

How do I get this to work? Currently using Powergrid version: 1.5.1
Similarly, how can I get the DatePicker to work with data from a relationship?

**Comment by luanfreitasdev** [comment]
Hi @JF-626164 

Nothing shows up ? To use with relationship, for example, categories.name

**Reply by JF-626164** [reply]
@luanfreitasdev, a text input box appears in the column head, but it doesn't filter.

**Reply by JF-626164** [reply]
It displays a "Select a Period" placeholder in the input box but when I click on it does nothing. I am using the Bootstrap5 theme

**Reply by luanfreitasdev** [reply]
Do you know if alpine is rendering correctly?. You can put it in the console window.Alpine and see the result.

**Reply by JF-626164** [reply]
I don't think it is. I put that "window.Alpine" command in the browser console and got back a "undefined"

**Reply by luanfreitasdev** [reply]
In the powergrid config already have the alpine cdn?

**Reply by JF-626164** [reply]
js_framework is set to alpinejs, the js_framework_cdn is set to 'alpinejs' => 'https://unpkg.com/alpinejs@3.x.x/dist/cdn.min.js'.

**Reply by JF-626164** [reply]
@luanfreitasdev so I finally figured out what the issue was. I wasn't calling @powerGridStyles in the layouts blade. Once I added that in the date picker works.

**Reply by luanfreitasdev** [reply]
☺️👏👏. That happens.
